.changeset/lucky-falcons-dress.md

```diff
@@ -0,0 +1,39 @@
+---
+'@backstage/cli': patch
```

Comment:
So nice to see a well written changeset! 👏 

As this is going to appear in both the cli and the create-app changelog currently I suggest that we split them out  so the cli have a text saying `Removed googleAnalyticsTrackingId configSchema.` or similar as there are really no changes required when bumping that specific package.

Reply:
I split `@backstage/cli` out into its own changeset. Thank you!